### PR TITLE
fix(Scripts/Ulduar): Hodir Door

### PR DESCRIPTION
**Pull request by Krath**
## CHANGES PROPOSED:
-  This commit fixes Hodir's doot not closing when the encounter starts. 

## TESTS PERFORMED:
- Tested ingame, works fine.

## HOW TO TEST THE CHANGES:
- Go to Ulduar and engage Hodir. The door should now be closing and opening when beating him or the raid dies.

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_hodir.cpp [hunk 1]
```diff
@@ -228,6 +228,8 @@ class boss_hodir : public CreatureScript
 
             if (pInstance && pInstance->GetData(TYPE_HODIR) != DONE)
                 pInstance->SetData(TYPE_HODIR, NOT_STARTED);
+            if (GameObject* go = me->FindNearestGameObject(GO_HODIR_FRONTDOOR, 300.0f))
+                    go->SetGoState(GO_STATE_ACTIVE);
```
Comment (Helias) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_hodir.cpp:232]: add brackets to the `if` statement, like:
```C++
            if (GameObject* go = me->FindNearestGameObject(GO_HODIR_FRONTDOOR, 300.0f))
           {
                    go->SetGoState(GO_STATE_ACTIVE);
            }
```

**Review by Helias — CHANGES_REQUESTED** [review]

**Comment by felwyn** [comment]
Checked! Fix is working!

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_hodir.cpp (hunk 1 above)
Comment (FrancescoBorzi) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_hodir.cpp]: done

**Comment by FrancescoBorzi** [comment]
I've fixed the codestyle. Next time please @Krath check the indentation and the brackets ;)

good job with this PR and thanks @felwyn for testing it!

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Krath** [comment]
Thank you all, I forgot to push the codestyle fix !

**Review by Helias — APPROVED** [review]

**Pull request merged**